bindings/python/test/test_numpy.py

```diff
@@ -50,7 +50,6 @@ def assert_numpy_equal(self, actual, expected):
             a = np.nan_to_num(actual[field])
             e = np.nan_to_num(expected[field])
             np.testing.assert_array_equal(a, e)
-            self.assertEqual(actual[field].dtype, expected[field].dtype)
```

Comment:
Why remove this assertion?

Reply:
Oops, forgot that the statement after the `if` I added was there before